src/js/actions/deviceActions.js

```diff
@@ -549,6 +536,24 @@ export const getAllDevicesByStatus = status => (dispatch, getState) => {
   return getAllDevices();
 };
 
+export const getDeviceAttributes = () => dispatch =>
+  GeneralApi.get(`${inventoryApiUrlV2}/filters/attributes`).then(({ data = [] }) => {
```

Comment:
We return up to 500 attributes here, not sure you need a limit in the UI... maybe not,

Reply:
a valid point indeed - we once had a limit, but soon after got feedback that there are some attributes missing 🙈... while the list of identity attributes is still limited, the filters might show all... the component we use there is definitely capable of a few hundred items though so we should be good 👌